.circleci/config.yml

```diff
@@ -329,14 +307,16 @@ commands:
           command: |
             if [[ "<< parameters.project >>" == "production" ]]; then
               export FEATURE_FLAGS_SHOW_RECOVERY_KEY_V2=true
+              export GREP="--grep="severity-1"
             else
               export FEATURE_FLAGS_SHOW_RECOVERY_KEY_V2=true
+              export GREP="--grep="severity-(1|2)"
```

Comment:
The quotes here look funny to me. It seems like maybe the end quote is missing, or perhaps the first quote isn't needed.